### DISCUSSION TITLE
Error zip: not a valid zip file

### DISCUSSION THREAD

**Discussion by ipchesnokovaep** [simulot/immich-go#201] [Q&A] [answered]
Hello!
Plz help
When i run immich-go error zip: not a valid zip file
takeout-*.zip size 2Gb
immich-go -server=http://1111:2283 -key=22 upload  -log-file=log.txt -create-albums -google-photos takeout-*.zip

**Comment by simulot** [comment, marked as answer]
Does the zip file open in your desktop?
Is you takeout file in the current directory? Otherwise give the full path

Check the command line if you have several parts in your takeout:
```sh
immich-go -server=http://1111:2283 -key=22 upload -log-file=log.txt -create-albums -google-photos takeout-*.zip
```

**Comment by ipchesnokovaep** [comment]
Thank you. 
And one of my takeout-*.zip files was corrupted. After redownloading file Immich-Go works fine
